Makefile

```diff
@@ -56,7 +56,7 @@ install-race:                   ## Install pmm-agent binary with race detector.
 TEST_PACKAGES ?= ./...
 TEST_FLAGS ?= -timeout=20s
 
-test:                           ## Run tests.
+test: install                   ## Run tests.
```

Comment:
I specifically removed it – tests are cached, but the final binary linking is not, and is is super-slow on macOS due to cgo.

Reply:
ok